### PR TITLE
Updated DevGuideValidation.md

### PR DESCRIPTION
**Pull request by invader35**
Updated DevGuideValidation.md to show that api is deprecated in gwt 2.8
See https://github.com/gwtproject/gwt/issues/9367 for further details

---
This change is [Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/183)

### CODE REVIEW

**Comment by invader35** [comment]
I signed it!

File: src/main/markdown/doc/latest/DevGuideValidation.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-Validation
+Validation (DEPRECATED)
 ===
 
+**Attention this API is deprecated in GWT 2.8 and will be removed in future versions**
```
Comment (tbroyer) [inline src/main/markdown/doc/latest/DevGuideValidation.md]: Not sure if we should say "removed", or "moved to a separate project".

Comment (invader35) [inline src/main/markdown/doc/latest/DevGuideValidation.md]: You could say, removed from gwt and add a note that there is going to be a spereate project for validation

**Comment by tbroyer** [comment]
Should we also remove the page from the menu? (`doc/latest/config.xml`) or at least mark it as deprecated there too, possibly moving it to a "deprecated" submenu.

**Comment by invader35** [comment]
Adding a deprecated submenu could make sense, if you want users from older versions finding that information more quickly.

**Comment by jnehlmeier** [comment]
I am with Gogtuk and would not call it deprecated at all: https://github.com/gwtproject/gwt/issues/9367#issuecomment-231327497

I would leave the title as is (without DEPRECATED) and adjust the "Attention" line below to

`CAUTION: Validation API is unmaintained and will be moved out of GWT SDK into a separate project after GWT 2.8.`

**Comment by invader35** [comment]
@jnehlmeier I changed it according to your comment

**Comment by invader35** [comment]
Anything new on this regard guys?

**Pull request merged**